modules/cluster-node-tuning-operator-default-profiles-set.adoc

```diff
@@ -89,3 +26,11 @@ spec:
   - profile: "openshift-node"
     priority: 40
 ----
+
+Starting with {product-title} 4.9, all OpenShift TuneD profiles are shipped with
+the TuneD package. You can use the oc exec command to view the contents of these profiles:
```

Comment:
Please enclose   `oc exec`  in the sentence in backticks.

Reply:
Oops, sorry.  Thank you, done.